deploy/helm/tracee/templates/tracee-policies.yaml

```diff
@@ -9,8 +9,9 @@ data:
   signatures.yaml: |-
     name: signature_events
     description: traces all signature events
-    defaultAction: log
-    scope:
+    defaultActions: 
+      - log
+    scopes:
```

Comment:
This should be kept as scope and not scopes (we have one scope, which is defined by several filters)

Reply:
ok